cmd/juju/crossmodel/list_test.go

```diff
@@ -76,18 +76,13 @@ func (s *ListSuite) TestListDirectories(c *gc.C) {
 		nil,
 		// Default format is tabular
 		`
-differentstore
-Application  Charm  Connected  Store           URL                                  Endpoint  Interface  Role
-zdiff-db2    db2    33         differentstore  differentstore:fred/model.zdiff-db2  log       http       provider
-                                                                                    mysql     db2        requirer
-myctrl
-Application  Charm  Connected  Store   URL                           Endpoint  Interface  Role
-hosted-db2   db2    0          myctrl  myctrl:fred/model.hosted-db2  log       http       provider
-                                                                     mysql     db2        requirer
-vendor
-Application  Charm  Connected  Store   URL                          Endpoint  Interface  Role
-adiff-db2    db2    23         vendor  vendor:fred/model.adiff-db2  log       http       provider
-                                                                    mysql     db2        requirer
+Application     Charm  Connected  Store           URL                                  Endpoint  Interface  Role
```

Comment:
Store is a general term. JAAS for example isn't a controller as such. Let's stick with controller for now and it can be bike shedded.

Reason for not having store in first column is that for the foreseeable future, there will only be one store, and application name is more important. We can adjust the tabular output if needed later.